action_name: calibration
kinematic_chain_base_link: yumi_base_link
loop_rate: 250
probe_arm:
  kdl_eef_frame: yumi_link_7_l
  gripping_frame: yumi_link_7_l
  has_ft_sensor: true
  sensor_frame: optodaq_l_sensor_link
  sensor_topic: /yumi/optodaq_l/ft_compensated
  sensor_calib:
    data: [1.0, 0, 0, 0, 0, 0,
           0, 1.0, 0, 0, 0, 0,
           0, 0, 1.0, 0, 0, 0,
           0, 0, 0, 1.0, 0, 0,
           0, 0, 0, 0, 1.0, 0,
           0, 0, 0, 0, 0, 1.0]

case_arm:
  kdl_eef_frame: yumi_link_7_r
  gripping_frame: r_gripping_point
  has_ft_sensor: true
  sensor_frame: optodaq_r_sensor_link
  sensor_topic: /yumi/optodaq_r/ft_compensated
  sensor_calib:
    data: [1.0, 0, 0, 0, 0, 0,
           0, 1.0, 0, 0, 0, 0,
           0, 0, 1.0, 0, 0, 0,
           0, 0, 0, 1.0, 0, 0,
           0, 0, 0, 0, 1.0, 0,
           0, 0, 0, 0, 0, 1.0]

kdl_manager:
  eps: 0.01
  max_tf_attempts: 10
  ikvel_solver: nso
  nso_weight: 40

calib:
  f: "-y"
  k_f: 0.01
  vd: 0.02
  wd: 0.04
  probe_tip_offset: 0.212
